docs/guides/basic-compositions/build-composition.md

```diff
@@ -0,0 +1,279 @@
+---
+title: Build your Composition
+tocHidden: true 
+weight: 260
+indent: true
+---
+
+# Build your Composition
+
+This is the third part of a four part tutorial to build an API for provisioning pre-configured VMs on top of the GCP Compute Engine service. If you have not already, you need to first [Build your XRD](build-xrd.md) first. In this part of the tutorial, you will build the Composition portion of your API.
+
+## Getting Started 
+
+The previous part of the tutorial had you build the first half of your API, the XRD, which defines the shape of your API. Now you are going to build the Composition, which is an implementation of the definition. Open the `composition.yaml` file created earlier. 
+
+Similar to the XRD, it is useful to bootstrap the file by copying in from a “known good” `composition.yaml` and pruning it back to the basics. Using the same repo as before, copy the PostgreSQL instance composition and repurpose it for your needs, found [here](https://raw.githubusercontent.com/upbound/platform-ref-gcp/main/package/database/postgres/composition.yaml), and paste it into your `composition.yaml` config. You could also go to the marketplace listing looked at earlier, scroll down and select the “XRDs” tab, and select the Composition titled [xpostgresqlinstances.gcp.platformref.upbound.io](https://marketplace.upbound.io/configurations/upbound/platform-ref-gcp/v0.3.0/compositions/xpostgresqlinstances.gcp.platformref.upbound.io/gcp.platformref.upbound.io/XPostgreSQLInstance), and copy the `.yaml` from there.

```

Comment:
Avoid "here" as the link text. https://developers.google.com/style/link-text#write-link-text